pkg/builddef/set.go

```diff
@@ -59,9 +59,9 @@ func (s *FilesystemBuildDefinitionSet) Get(ctx context.Context, t rebuild.Target
 }
 
 func (s *FilesystemBuildDefinitionSet) Path(ctx context.Context, t rebuild.Target) (string, error) {
-	definitions := rebuild.NewFilesystemAssetStore(s.fs)
-	_, pth, err := definitions.Reader(ctx, rebuild.Asset{Type: rebuild.BuildDef, Target: t})
-	return pth, err
+	defs := rebuild.NewFilesystemAssetStore(s.fs)
+	url := defs.URL(rebuild.Asset{Type: rebuild.BuildDef, Target: t})
+	return url.Path, nil
```

Comment:
Why not return in one line?

Reply:
figured it was marginally easier to read this way. that said, i think it might be a good idea to revisit this API after this change.